src/EnergyPlus/SimAirServingZones.cc

```diff
@@ -6009,7 +6059,7 @@ namespace SimAirServingZones {
 
 			// write predefined standard 62.1 report data
 			for ( AirLoopNum = 1; AirLoopNum <= NumPrimaryAirSys; ++AirLoopNum ) {
-				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {
+//				if ( FinalSysSizing( AirLoopNum ).SystemOAMethod == SOAM_VRP ) {  // commented line allows ZoneSum method to report tables to the html file
```

Comment:
I'll also add that I don't like the data shown without multipliers since it's a simple calculation to show the actual data. I think the note was added instead of correcting the report.